docs/frontend/developer-guide.md

```diff
@@ -120,7 +120,11 @@ Our apps use chained middlewares to improves code readability, and maintainabili
 
 The Registration1 and Registration code is organized into sub-folders based on a identity provider, an application role, and dashboard folder, or just dashboard folder for routes available for authenticated users without an authorization role. As mentioned, the middleware dynamically rewrites the request URL based on the Auth.js JWT properties of identity_provider and user role so to match our sub-folder structure. So route URL segments such as registration1 `http://localhost:3000/dashboard/operations` would get mapped to nested folder `bciers/apps/registration1/app/(authenticated)/bceidbusiness/industry_user/dashboard/operations' for an authenticated industry user.
 
-For our multi-zone apps, the dashboard app manages the main domain and rewites request to the appropriate zone as required. The dashboard links are configured in `bc_obps/common/fixtures/dashboard/{identity-provider}` and use the `{project/}` pre-fix. So route URL to registration operations would be `http://localhost:3000/registration/dashboard/operations`
+For our multi-zone apps, the dashboard app manages the main domain and rewites request to the appropriate zone as defined in `bciers/apps/dashboard/next.config.js` and `bciers/apps/dashboard/middlewares/withAuthorizationDashboard.ts`. The dashboard links are configured in `bc_obps/common/fixtures/dashboard/{identity-provider}` and use the `{project/}` pre-fix. So route URL to registration operations would be `http://localhost:3000/registration/dashboard/operations`
+
+### Folder Structure & Dynamic Breadcrumbs
+
+Shared component `bciers/libs/components/src/navigation/Bread.tsx` builds the breadcrumbs dynamicaly from the folder structure. For folders that should not be display in the breadcrumbs we can use `Route groups`, a folder in parenthesis e.g. (authentication), which are not included in the route's URL path, or we can use conditional logic in the component. For example, we handle UUID breadcrumbs from dynamic folders by passing the name associated with the UUID record in the querystring. The Bread component then uses `translateNumericPart` for UUID segments and finds the correct querstring parameter rendering the name instead of the number. The pattern used for the passed querystring parameter should be `{name-of-preceeding-folder}Title`. As example, the `OperationDataGridPage` is the page displayed from route `bciers/apps/administration/app/bceidbusiness/industry_user_admin/operations` and the `View Details` link passes parameter `?operationsTitle` for route `bciers/apps/administration/app/bceidbusiness/industry_user_admin/operations/[operatorId]`.
```

Comment:
a couple typos: ```suggestion
Shared component `bciers/libs/components/src/navigation/Bread.tsx` builds the breadcrumbs dynamically from the folder structure. For folders that should not be displayed in the breadcrumbs we can use `Route groups`, a folder in parenthesis e.g. (authentication), which are not included in the route's URL path, or we can use conditional logic in the component. For example, we handle UUID breadcrumbs from dynamic folders by passing the name associated with the UUID record in the querystring. The Bread component then uses `translateNumericPart` for UUID segments and finds the correct querystring parameter rendering the name instead of the number. The pattern used for the passed querystring parameter should be `{name-of-preceeding-folder}Title`. As example, the `OperationDataGridPage` is the page displayed from route `bciers/apps/administration/app/bceidbusiness/industry_user_admin/operations` and the `View Details` link passes parameter `?operationsTitle` for route `bciers/apps/administration/app/bceidbusiness/industry_user_admin/operations/[operatorId]`.
```

Reply:
Also, I don't totally understand this line: `or we can use conditional logic in the component.` In the breadcrumbs component, or in a different component? I didn't see conditional logic in the PR, the breadcrumbs component seems to work out of the box

Reply:
@BCerki 

Thank-you for the editing of the documentation, I will make corrections.

An example of what I am calling 'conditional logic` is the function which checks if a segment is a UUID and if yes then does this else that...see `function translateNumericPart`